tinyiiod.c

```diff
@@ -32,6 +33,7 @@ struct tinyiiod * tinyiiod_create(const char *xml,
 	if (!iiod)
 		return NULL;
 
+	iiod->iiod_buf = malloc(IIOD_BUFFER_SIZE);
```

Comment:
If iiod_buf will be renamed, instead of iiod->iiod_buf we will simply have iiod->buf.